doc/rfcs/003-quickly-collaborate-with-coworkers-and-friends.md

```diff
@@ -0,0 +1,146 @@
+
+When your past collaborators have Atom open and they're signed into Teletype, they appear as "online" in your list of past collaborators. You can select a past collaborator (an "invitee" in this context) and invite them to join your portal. If you're not already hosting a portal, Teletype automatically creates a portal for your workspace.
+
+Inside the invitee's Atom instance, Teletype informs the invitee that you have invited them to join your portal, and Teletype asks them if they want to join it. If the invitee chooses to join the portal, they see a notification confirming that they've joined the portal. If the invitee declines to join the portal, Teletype notifies the host that the guest declined the invitation.
+
+### Prevent invitations from a past collaborator
+
+You can remove anyone from your list of past collaborators. Once you've removed someone from your list of past collaborators, you will appear as "offline" in their list of past collaborators, and they can no longer invite you to their portal from directly within Atom.
+
+If you later decide that you want to collaborate with them again, you can send them a URL (via Slack, IRC, etc.) to join your portal, or you can ask them to send you a URL (via Slack, IRC, etc.) so that you can join their portal. Once you collaborate in a portal together again, Teletype adds them back to your list of past collaborators, and they can once again see your online/offline status in their list of past collaborators.
+
+## Out of scope
+
+In the interest of getting the highest impact functionality in users' hands as quickly as possible and then iterating based on real-world feedback, the following functionality is out of scope for this RFC, but may be addressed in follow-up releases.
+
+- Changing your online/offline status
+    - Setting your status to offline. (In the meantime, you can sign out of Teletype or disable Teletype in Atom's package settings when you want to use Atom while not accepting invitations from your past collaborators.)
+    - Setting your status to "away" or "busy"
+- UX enhancements to the list of past collaborators
+    - Changing the sort order for your list of past collaborators (e.g., sort by how recently you've collaborated, sort or group by online/offline status)
+    - Limiting the size of your list of past collaborators. (In the meantime, you can remove past collaborators to reduce the size of the list.)
+    - Filtering your list of past collaborators
+    - Marking certain collaborators as "favorites"
+- Offering to install Teletype if you click on a portal URL and don't yet have Teletype installed ([#220](https://github.com/atom/teletype/issues/220))
+- Selecting a past collaborator and asking to join their portal
+- Selecting multiple past collaborators and inviting them to your portal simultaneously. (In the meantime, you can select a collaborator and invite them, then select another collaborator and invite them, and so on.)
```

Comment:
That would be nice for use cases like recurring stand-ups or meetings. But it's probably easiest to just share the URL when there are many people.

Here another idea:

Past collaborators shown as group. It's pretty much the same as Slack's DMs list where it can be just a single person or a group (Don and Amanda):

![image](https://user-images.githubusercontent.com/378023/37577133-72c9fb4e-2b73-11e8-96f6-d0986b565553.png)

That way a host can easily continue a collaboration with multiple people. Like "let's continue tomorrow at 10am". Then the next day, the host can invite the whole group at once. Although this only makes sense if the list is sorted by most recent. Otherwise you wouldn't know which person gets used for an alphabetic list.

Reply:
> Here another idea: Past collaborators shown as group. 

I dig that as a potential future enhancement.